3IPs/3ip-2.md

```diff
@@ -58,6 +55,41 @@ This function returns an array of posts made in the thread. It's also possible t
 const posts = thread.getPosts()
 ```
 
+**Delete a thread post:**
+Attempts to delete a post with given id. Deletion completes if the user has the capabilities in the access controller to do so.
+
+```js
+await deletePost(<id>)
+```
+
+**Add a moderator to a thread:**
+Attempts to add a moderator to a thread. Returns id if sucessful, otherwise false.
+
+```js
+await addMod(<id>)
```

Comment:
Why use the abbreviated version? Why not `addModerator`?

Reply:
I have no preference, we can use `addModerator`

Reply:
Ok I think `addModerator` is clearer!

Reply:
👍